lib/fir/key_command/ctrl_u_command.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+# encoding: UTF-8
+
+require_relative './key_command'
+
+class Fir
+  class CtrlUCommand < KeyCommand
+    def self.character_regex
+      /^\u0015$/
+    end
+
+    def execute_hook(new_state)
+      unless state.blank?
```

Comment:
could we collapse this unless and if into a single expression? `if !state.blank? && state.cursor.x.positive?`